README.md

```diff
@@ -14,25 +15,72 @@
 
 ## Installation
 
+- `git clone https://github.com/NewTec-GmbH/lobster-doxygen.git` to clone repository.
+- `cd lobster-doxygen` to go to root directory.
+- It is recommended to use a virtual environment:
+  - In VS-Code: `Ctrl+Shift+P` type **Python: Create Environment**: select `Venv`.
+    - When asked, select the Python version you want to work with, based on the versions installed in your machine.
+    - When asked, select `dev` to be installed. This will set-up the development environment, including all the tools used in this template.
+    - In the background, VS Code will install all the dependencies. This may take some time.
+    - To activate the virtual environment, close all terminal panels inside of VS-Code.
+      You can double check if the virtual environment is active, e.g. by `pip -V` the displayed path should point to your virtual environment.
+  - In PowerShell:
+    - `python -m venv .venv` to create the virtual environment.
+    - `.venv\Scripts\Activate.ps1` to activate it.
+    - `pip install -e .` to install required packages.
+    - `pip install -e .[dev]` to install additional packages for development.
+  - Under Linux / MacOS:
+    - `python -m venv .venv` to create the virtual environment.
+    - `source .venv/bin/activate` to activate it.
+    - `pip install -e .` to install required packages.
+    - `pip install -e .[dev]` to install additional packages for development.
+  For more details about handling Venv, see [Python venv: How To Create, Activate, Deactivate, And Delete](https://python.land/virtual-environments/virtualenv#Python_venv_activation)
+
+## Usage
+
+lobster-doxygen is a command line application that is configured via command line arguments.
+
 ```bash
-git clone https://github.com/NewTec-GmbH/lobster-doxygen.git
-cd lobster_doxygen 
-pip install .
+usage: lobster-doxygen [-h] [-o OUTPUT] [-v] doxygen_folder
+
+Script to generate the lobster common interchange format from a doxygen XML output.
+
+positional arguments:
+  doxygen_folder       Path to the doxygen XML output folder.
+
+options:
+  -h, --help           show this help message and exit
+  -o, --output OUTPUT  Output file name.
+  -v, --verbose        Enable verbose output.
 ```
 
-## Usage
+## Doxygen XML files
+
+In order to feed lobster-doxygen with the correct data, Doxygen needs to be configured.
+For the requirement annotation the `Doxyfile` needs the following aliases:
+
+```bash
+ALIASES                = "implements{1}=@xrefitem implements \"Implements\" \"Requirement Traceability\" Requirement: \1" \
+                         "justification{1}=@xrefitem justified \"Justified\" \"Justification Overview\" Justification: \1"
+```
 
->TODO
+To enable the XML output:
 
 ```bash
-lobster_doxygen [-h] [-v] {command} {command_options}
+GENERATE_XML           = YES
 ```
 
-Detailed descriptions of arguments
+Once doxygen has run successfully, the `xml` directory can be set as the `doxygen_folder` in the application.
```

Comment:
Note, that for regulatory projects doxygen must be configured to export everything. Otherwise there might be e.g. private functions, which will never be part of the traceability. You can check the doxyfile against a default doxyfile to see all of the changes. Shows them as example in case of a C++ project.